dist/flatpak/io.github.input_leap.InputLeap.yml

```diff
@@ -9,35 +9,58 @@
 # Note: mDNS (foo.local) does NOT work, so server.name needs to be a FQDN or an ip address
 # https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1013
 
-app-id: com.github.inputleap.InputLeap
-runtime: org.gnome.Platform
-runtime-version: '42'
-sdk: org.gnome.Sdk
-command: input-leap-flatpak
+app-id: io.github.input_leap.InputLeap
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: input-leap
 
 finish-args:
     # We want to talk to the network
     - --share=network
     # Need to read our config from somewhere
     - --filesystem=xdg-config/input-leap
```

Comment:
We save our config in `~/.config/InputLeap` not `~/.config/input-leap`

Reply:
updated